### DISCUSSION TITLE
Gstreamer webrtc streaming

### DISCUSSION THREAD

**Discussion by VanderBieu** [bluenviron/mediamtx#3572] [Questions]
### Question

Sample code 
`gst-launch-1.0 videotestsrc \
! video/x-raw,width=1920,height=1080,format=I420 \
! x264enc speed-preset=ultrafast bitrate=2000 \
! video/x-h264,profile=baseline \
! whipclientsink signaller::whip-endpoint=http://localhost:8889/mystream/whip`

not working. Log indicates that this stream never reached the mediamtx server. How to make this work?
My platform is windows-amd64, my gstreamer version is 1.24.4, mediamtx version 1.8.4

**Comment by aler9** [comment]
answered in https://github.com/bluenviron/mediamtx/issues/3573
